### ISSUE TITLE
Typing on the windows 11 search bar

### ISSUE THREAD

**Issue by murasu** [jrywu/DIME#80]
Hi I just compiled and installed the dll on my Windows11 machine. While I am able to type in apps like Notepad and Word, I am not able to type in the system search bar. I don't know of other UWP apps to try. Is this a known issue?

**Comment by jrywu** [comment]
No. This codebase supposedly works well even in UWP apps.  The latest windows 11 search bar is using UWP rendering and should  work well.
